src/main/java/com/endava/Ticket_Management_System/model/Venue.java

```diff
@@ -0,0 +1,65 @@
+package com.endava.Ticket_Management_System.model;
+
+import jakarta.persistence.*;
+
+import java.io.Serializable;
+
+@Entity
+@Table(name = "Venue")
+public class Venue implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int venueID;
+
+    @Column(name = "location")
+    private String location;
+
+    @Column(name = "type")
+    private String type;
+
+    @Column(name = "capacity")
+    private int capacity;
+
+    public Venue() {
```

Comment:
Why didn't you use lombok for constructors, getter and setters? I see you have it as a dependency